packages/mobile_app/lib/services/ads.dart

```diff
@@ -0,0 +1,35 @@
+part of 'static_provider.dart';
+
+RewardedAd getCardAd() => RewardedAd(
+      adUnitId: 'ca-app-pub-3940256099942544/5354046379',
+      request: AdRequest(),
+      listener: AdListener(
+        // Called when an ad is successfully received.
+        onAdLoaded: (Ad ad) {
+          if (StaticProvider.debug.isDebug) print('Ad loaded.');
+        },
+        // Called when an ad request failed.
+        onAdFailedToLoad: (Ad ad, LoadAdError error) {
+          ad.dispose();
+          if (StaticProvider.debug.isDebug) print('Ad failed to load: $error');
```

Comment:
I think we shoild call `cards.whenAdsWatched();` as well, since that is not user's fault that we cannot show the ad